test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -883,7 +883,7 @@ class test_ucp_tag_offload_stats_gpu : public test_ucp_tag_offload_stats {
 };
 
 UCS_TEST_P(test_ucp_tag_offload_stats_gpu, block_gpu_no_gpu_direct,
-           "TM_THRESH=128")
+           "TM_THRESH=1")
```

Comment:
is it possible to set TM_THRESH=0?

Reply:
yes, it is, but zero-size messages will not be offloaded anyway